docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents.TextLabel` namespace contains the class.

```

Comment:
The `TextLabel` class provides a control that renders a short text string. The `TextLabel` class is present in the  `Tizen.NUI.BaseComponents.TextLabel` namespace.

To maintain consistency across pages. 
 https://developer.tizen.org/dev-guide/2.4/org.tizen.ui.practices/html/native/dali/textlabel_n.htm

Reply:
Please refer https://github.com/Samsung/tizen-docs/pull/1009#discussion_r386171020

Reply:
I tried to align to style in the following latest doc:
https://developer.tizen.org/development/guides/.net-application/natural-user-interface/ui-components/textlabel (updated on 30 Oct 2019)

And also kept the following comment in consideration too while providing the sentence:
#1009 (comment) .

Kindly suggest a better rephrase if my suggested rephrase is not good.

Reply:
@JoogabYun Thanks, here updates have been made as per @journey2w 's suggestion.
Line: 14, 16
https://github.com/Samsung/tizen-docs/pull/1009#discussion_r377380016